bpf/kmesh/workload/cgroup_sock.c

```diff
@@ -49,12 +49,55 @@ static inline bool check_kmesh_enabled(struct bpf_sock_addr *ctx)
     return bpf_map_lookup_elem(&map_of_manager, &cookie);
 }
 
+static inline bool check_bypass_enabled(struct bpf_sock_addr *ctx)
+{
+    __u64 cookie = bpf_get_netns_cookie(ctx);
+    return bpf_map_lookup_elem(&map_of_bypass, &cookie);
+}
+
+static inline void record_bypass_netns_cookie(struct bpf_sock_addr *ctx)
```

Comment:
The logic seems to be consistent with record_netns_cookie? Can they be combined into one function?

Reply:
> The logic seems to be consistent with record_netns_cookie? Can they be combined into one function?

I've extract the function